lib/src/node/compile.dart

```diff
@@ -239,6 +265,7 @@ List<AsyncCallable> _parseFunctions(Object? functions, {bool asynch = false}) {
       late Callable callable;
       callable = Callable.fromSignature(signature, (arguments) {
         var result = (callback as Function)(toJSArray(arguments));
+        result = simplify(result);
```

Comment:
Style nit:

```suggestion
        var result = simplify((callback as Function)(toJSArray(arguments)));
```

Generally good to avoid reassigning variables if possible.